docs/source/conf.py

```diff
@@ -189,7 +189,7 @@
 # #html_style = 'default.css'
 # #html_theme = 'classic'
 
-# Useful aliases to avoid repeating long URLs. 
+# Useful aliases to avoid repeating long URLs.
```

Comment:
This was my IDE doing... as I was about to revert this, I figured that this is probably a good drive-by, so I kept it.